ieeeuoft/hackathon-template
# Updating chart data with MakeUofT 2023 and NewHacks 2023

## Overview

- Resolves [#184](https://ieeeuoft.atlassian.net/browse/IEEE-184)
- Adding the MakeUofT 2023 and NewHacks 2023 application data to the charts on the admin dashboard
<img width="911" alt="image" src="https://github.com/ieeeuoft/hackathon-template/assets/54194957/3592532e-e759-4754-9ade-ab0b99c07e9b">
<img width="508" alt="image" src="https://github.com/ieeeuoft/hackathon-template/assets/54194957/eee3f93c-826b-4618-91a0-34a97409ef75">


## Unit Tests Created

- None because these are changes to the admin site


## Steps to QA

- go to http://localhost:8000/admin/registration/application/ and check out the graphs
